src/components/Page/fonts/fonts.css

```diff
@@ -0,0 +1,119 @@
+/* Prevent mobile browsers from changing font-size */
+* {
+  -webkit-text-size-adjust: none;
+  text-size-adjust: none;
+}
+
+@font-face {
+  font-family: 'BrandonGrotesque';
+  src: local('Brandon Grotesque Light'), local('BrandonGrotesque-Light'),
+    url('./BrandonGrotesque-Light.woff2') format('woff2'),
```

Comment:
just curious - do we need both woff2 and woff?

Reply:
Unfortunately yes. IE 11 doesn't support woff2... https://caniuse.com/#search=woff2

Reply:
got it ... but do we need the third type - otf? or what was that name? can we just keep otf?